ivy/functional/frontends/jax/numpy/name_space_functions.py

```diff
@@ -94,6 +94,10 @@ def einsum(*operands, out=None, optimize=None, precision=None, _use_xeinsum=Fals
     return ivy.einsum(equation=optimize, *operands, out=out)
 
 
+def floor_divide(x1, x2):
```

Comment:
feel free to create an issue and close it for the Todo list tracking since you've done this function here :)